apps/mull-api/axiosConfig.ts

```diff
@@ -0,0 +1,6 @@
+import axios from 'axios';
+const axiosInstance = axios.create({
+  baseURL: `https://app.geocodeapi.io/api/v1/autocomplete?apikey=${process.env.GEOCODE_KEY}&text=test_location&size=5`,
```

Comment:
The API doesn't seem to work too well, I put a park near my house and it couldn't find it. Could we look into a better API? Or is it another issue?

Very quick googling and I found this API which found the park near my house https://community.algolia.com/places/
Unfortunately they're shutting down in 2022, if we can't use this maybe there's others?